### DISCUSSION TITLE
mouse stops moving in osu!(lazer)

### DISCUSSION THREAD

**Discussion by Ithoo235** [ppy/osu#24282] [Q&A] [answered]
i recently switched to osu!(lazer) and at seemingly random times my mouse stops moving. it can happen from multiple times a beatmap to once every 5 beatmaps. i have tried to toggle high precision mouse but it happens either way. this only happens in lazer and nowhere else so my mouse is not the problem.

https://cdn.discordapp.com/attachments/817404219492794448/1130485823641485402/osuwierdbugidk.mp4
here is video proof of it happening(sorry ab it being a download idk how to fix that)

**Comment by peppy** [comment]
- Can you try setting the game to single thread mode?
- Can you tell us what operating system you're on?
- Can you take another video of it happening after pressing Ctrl+F11 twice (should bring up some graphs which will help us understand the issue)?

**Reply by Ithoo235** [reply]
https://cdn.discordapp.com/attachments/817404219492794448/1130836449357475951/osubuggg.mp4
im gonna try singlethreaded next.
as for the OS im using windows 11

**Reply by Ithoo235** [reply]
https://cdn.discordapp.com/attachments/817404219492794448/1130843390867669032/singlethread1.mp4
https://cdn.discordapp.com/attachments/817404219492794448/1130843392327307285/singlethread2.mp4
after i switched to singlethread the problem got converted(i think) to more rapid stuff. the one where the mouse stopped moving for 1-2 seconds is gone but there are much more rapid ones that lasts for 0.1-0.2 seconds. they show up around every minute

**Reply by peppy** [reply, marked as answer]
The graphs are reporting that this is something else on your PC likely hooking input. A starting point would be to figure out which program is causing it by closing things down one by one until it stops.

**Reply by Ithoo235** [reply]
it was my browser(opera gx)
i think the problem is gone now when i close my browser.
ty for your time

**Reply by peppy** [reply]
Thanks for checking. I'm hoping we can replicate this and fix the issue from our end.

Can you try some other tests:

- Does it happen with no tabs open in the browser?
- Does it happen when the browser is minimised?
- Do you use multiple displays / was the browser on another screen?
- Any extensions or anything else you noticed?

**Reply by Ithoo235** [reply]
i only have 1 screen and i always just tabbed back to osu.
 i did some tests and the input stuff from what i have found only happens after starting your pc and goes away in 3 hours. also goes away if i close the browser.
i closed all the tabs in my browser and it eased down a little but the browser itself seems to be the problem
